### PR TITLE
Add some additional information to the top level readme for clarity

### PR DESCRIPTION
**Pull request by mprorock**
Additions to the top level repository readme so that it is clear what lives where along with stated goals for the repository.  Just trying to make this repo a little easier to approach if you haven't been living in the ccg calls/mailing lists/etc.

The info I have added is based on reading the repo, previous issues, mailing list discussions, etc. but is based on my understanding, so I may have missed something.  Please add additional commentary on items that I may need to add or reword.

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
-on the
-[public-credentials](http://lists.w3.org/Archives/Public/public-credentials/)
-mailing list as well.
+The VC HTTP API repository contains a standard API specification for constructing and verifying objects which conform to the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/) specification, along with documentation, integration and compatability tests, and related assets for the test and integration process.
+
+### Structure
+
+The vc-http-api repository is structured with two main sections:
+
+- [docs](./docs/) which contains the VC HTTP API spec itself in [OpenAPI](https://swagger.io/specification/) form as well as related documentation to assist in building and testing a compliant implementaiton of the API spec
```
Comment (TallTed) [inline README.md:9]:
```suggestion
- [docs](./docs/) which contains the VC HTTP API spec itself in [OpenAPI](https://swagger.io/specification/) form as well as related documentation to assist in building and testing a compliant implementation of the API spec
```

**Comment by mprorock** [comment]
@msporny @peacekeeper - any requested changes?

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
> @msporny @peacekeeper - any requested changes?

LGTM.

**Review by mavarley — APPROVED** [review]

**Comment by mprorock** [comment]
@msporny @OR13 @peacekeeper - can we go ahead and get this merged in?

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**